addons/core/translations/resources/en-us.yaml

```diff
@@ -420,9 +420,40 @@ worker:
   title: Worker
   title_plural: Workers
   description: Workers are the server components that perform the session handling.
+  actions:
+    register: Register Worker
   messages:
     welcome:
       title: Welcome to Workers
   table:
     session_count: Session Count
     ip_address: IP Address
+  form:
+    cluster_id:
+      label: Boundary Cluster ID
+      help: The cluster ID is the UUID in the HCP Boundary Controller URL
+    ip_address:
+      label: IP Address
+      help: The public IP address of the server
+    config_file_path:
+      label: Config file path
+      help: The path on your server where you will store your configuration file
+    worker_tags:
+      label: Worker Tags
+      help: Boundary can use tags to define key-value pairs which targets can use to determine where they should route connections
+    steps:
+      1:
+        title: 1. Create a config file
+        create_directory: Create a configuration file in a new directory on your server
+        create_config: 'Open the file with a text editor, such as Vi. Paste the following configuration into the worker config file:'
+        save_config: Save the configuration in your `pki-worker.hcl` file.
```

Comment:
I kind of wanted to wrap the `pkie-worker.hcl` words in a `<code></code>` block but wasn't sure how to accomplish this cleanly without using the `htmlSafe` [option](https://ember-intl.github.io/ember-intl/docs/helpers/t#format-html-message).

Any recommendations, are we opposed to using this for wrapping HTML that doesn't have user input? I know that it's not recommended in general to render arbitrary HTML but in this case it's just HTML tags we're passing along. 

Let me know your thoughts.